From c49781bcea6367dd17e9245783a14964fd45e555 Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Tue, 29 May 2018 06:49:23 +0200
Subject: [PATCH] v2v: fix "rhevexp" typo in documentation

Spotted by Ming Xie.

(cherry picked from commit c021ac5e64053052a392762aaff9c8ed73fc2082)
---
 v2v/virt-v2v.pod | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/v2v/virt-v2v.pod b/v2v/virt-v2v.pod
index 62b726b8f..806225be0 100644
--- a/v2v/virt-v2v.pod
+++ b/v2v/virt-v2v.pod
@@ -601,7 +601,7 @@ Currently there are two possible flavours:
 
 =over 4
 
-=item rhevexp
+=item rhvexp
 
 The OVF format used in RHV export storage domain.
 
@@ -611,7 +611,7 @@ The OVF format understood by oVirt REST API.
 
 =back
 
-For backward compatibility the default is I<rhevexp>, but this may change in
+For backward compatibility the default is I<rhvexp>, but this may change in
 the future.
 
 =item B<-op> file
-- 
2.17.2

